jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -305,24 +316,23 @@ private void enableGracefulShutdownMode()
     public void expireConference(Conference conference)
     {
         String id = conference.getID();
-        boolean expireConference;
 
         synchronized (conferencesById)
         {
             if (conference.equals(conferencesById.get(id)))
             {
                 conferencesById.remove(id);
-                expireConference = true;
-            }
-            else
-            {
-                expireConference = false;
+                conference.expire();
+                if (conference.includeInStatistics())
```

Comment:
I'm feeling like `includeInStatistics` is being processed at the wrong level here - there's nothing about the `conferenceExpired` event that's intrinsically related to statistics.  I'd say that either it should be processed in the handler, or else the statistics-related handler shouldn't be installed for non-`includeInStatistics` conferences.

Reply:
I agree that I find this aspect (and the general special-casing for 'health check conferences') to be awkward in general.

>  I'd say that either it should be processed in the handler

I can, but I think there's an argument that this flag is already more exposed than we want it, and the stat handler shouldn't have to care about this.  Since the conference really is basically an "internal, videobridge health check tool", I thought it made sense that the bridge be the one who "hides" it.

> or else the statistics-related handler shouldn't be installed for non-includeInStatistics conferences.

The subscription is done at the "Videobridge events" level, not per conference so I couldn't do this (the way it's currently implemented, at least).

Reply:
I'm not necessarily against changing it (of your suggestions, I think the handler doing it probably makes the most sense), but if my reasoning is convincing I'll leave it.

Reply:
If we're going to leave it, there should at least be a comment at this point in the code that at present this event is only used for statistics, and if it were to be used for something else the code would need to be changed...

Reply:
Ended up moving it to the handler.

Reply:
Changed my mind on this again: I think this is something that is internal to Videobridge and we wouldn't ever expose this conference as part of the events interface.  I do agree the health check conference stuff could use some cleanup.